### PR TITLE
Enhancement: AnyString and Empty

### PR DESCRIPTION
**Pull request by tigh-latte**
Added AnyString func to check lists of strings without having to compile regex or define an inline func.

Added Empty func which just inverses NotEmpty.

Use IsZero for NotEmpty.

Added unit tests.

### CODE REVIEW

File: functions.go [hunk 1]
```diff
@@ -282,6 +264,22 @@ func NotEmpty(v interface{}) ValidationFunc {
 	}
 }
 
+// Empty will ensure that a value, val, is not empty.
```
Comment (theflyingcodr) [inline functions.go]: Should this say ` Empty will ensure that a value, val, is empty.`

**Review by theflyingcodr — CHANGES_REQUESTED** [review]

File: functions.go (hunk 1 above)
Comment (tigh-latte) [reply functions.go]: So it should

Comment (tigh-latte) [reply functions.go]: at's us nai

**Review by theflyingcodr — APPROVED** [review]
LGTM

**Pull request merged**